akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -359,36 +379,30 @@
     <profiles>
 
         <profile>
-            <!-- run Integration Tests in src/it/java with `mvn verify -Pit`-->
             <id>it</id>
             <build>
                 <plugins>
                     <plugin>
-                        <!-- run *IntegrationTest with failsafe -->
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-failsafe-plugin</artifactId>
-                        <version>${maven-failsafe-plugin.version}</version>
+                        <groupId>org.codehaus.gmaven</groupId>
+                        <artifactId>groovy-maven-plugin</artifactId>
+                        <version>2.1.1</version>
                         <executions>
                             <execution>
+                                <phase>integration-test</phase>
                                 <goals>
-                                    <goal>integration-test</goal>
-                                    <goal>verify</goal>
+                                    <goal>execute</goal>
                                 </goals>
                                 <configuration>
-                                    <includes>
-                                        <include>**/*IntegrationTest</include>
-                                    </includes>
-                                    <argLine>-Dlogback.configurationFile=${logback.configurationFile} -Dakka.javasdk.dev-mode.project-artifact-id=${project.artifactId}</argLine>
+                                    <source>
+                                        log.warn('This profile is deprecated. It will be removed in future versions. Integration tests only need `mvn verify` to run.')
```

Comment:
This is printed out at the end of the execution and it's not clear with profile we are talking about. 

So this text could be: 

```suggestion
                                        log.warn('The 'it' profile is deprecated. It will be removed in future versions. Integration tests only need `mvn integration-test` to run.')
```

Reply:
I thought the same in a first instance, then I though that since the user has to pass the profile when executing the command might be nicer not to add it here. But if you also thought it's missing I guess it's just better to add it.